src/question/views.py

```diff
@@ -67,7 +68,51 @@ class CommentViewSet(Votable):
 
 class QuestionViewSet(
     mixins.RetrieveModelMixin,
+    mixins.DestroyModelMixin,
+    mixins.UpdateModelMixin,
     Votable
 ):
     model = models.Question
     serializer_class = serializers.QuestionSerializer
+    permission_classes = [QuestionOwnerOrSafeMethod]
+
+    @action(detail=True, methods=['post'], permission_classes=[permissions.IsAuthenticated])
+    def answers(self, request, pk):
+        question = get_object_or_404(self.model, pk=pk)
+
+        user = request.user
+        user_profile = Profile.objects.get(user=user)
+        try:
+            text = request.data['text']
+            show_username = request.data['show_username']
+        except KeyError:
+            return HttpResponse(status=status.HTTP_400_BAD_REQUEST)
+
+        answer = models.Answer.objects.create(
+            text=text,
+            show_username=show_username,
+            votes=Votes.objects.create(),
+            owner=user_profile,
+            parent=question,
+            is_accepted=False
+        )
+
+        serializer = serializers.AnswerSerializer(answer)
+        return HttpResponse(serializer.data, status=status.HTTP_201_CREATED)
+
+    def update(self, request, *args, **kwargs):
+        # only the 'title', text', 'tags' can be updated
```

Comment:
Because this is just the default implementation it could be replaced by a call to `self.update(request, partial=True)`